src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -60,9 +60,14 @@
         PREVIEW
     }
 
+    interface Type {
+//        String ARTICLE = "http://www.ft.com/ontology/content/Article";
```

Comment:
I'd rather remove the commented lines completely, as I think they are slightly confusing (mainly the `CONTENT` one, even if I get that it would be a safe default).